experimental/PropertyDDS/packages/property-dds/src/propertyTree.ts

```diff
@@ -377,7 +376,9 @@ export class SharedPropertyTree extends SharedObject {
 			message.type === MessageType.Operation &&
 			message.sequenceNumber > this.skipSequenceNumber
 		) {
-			const change: IPropertyTreeMessage = this.decodeMessage(cloneDeep(message.contents));
+			const change: IPropertyTreeMessage = this.decodeMessage(
```

Comment:
the typing on lodash is evidently more strict than on fastest-json-copy, hence this cast (`message.contents` has type `unknown` and lodash's typing is the expected `cloneDeep: <T>(x: T): T`)